isotope/service/pkg/srv/handler.go

```diff
@@ -54,6 +56,13 @@ func (h Handler) ServeHTTP(writer http.ResponseWriter, request *http.Request) {
 		prometheus.RecordResponseSent(duration, len(h.responsePayload), status)
 	}
 
+	statusCode := <-h.StatusTicker.StatusChan
```

Comment:
Seems like we can just use an atomic integer to simplify the channel stuff?

Reply:
Good one. Didn't even think about it, I'll update the PR!